graviti/openapi/draft.py

```diff
@@ -0,0 +1,239 @@
+    """Execute the OpenAPI `GET /v1/datasets/{dataset_name}/drafts/{draft_number}`.
+
+    Arguments:
+        url: The URL of the graviti website.
+        access_key: User's access key.
+        dataset_name: Name of the dataset, unique for a user.
+        draft_number: Number of the draft.
+
+    Returns:
+        The response of OpenAPI.
+
+    Examples:
+        >>> get_draft(
+        ...     "https://graviti.com/",
+        ...     "ACCESSKEY-********",
+        ...     "MNIST",
+        ...     2,
+        ... )
+        {
+           "number": 2,
+           "title": "branch-2",
+           "description": "",
+           "branch_name": "main",
+           "state": "OPEN",
+           "parent_commit_id": "85c57a7f03804ccc906632248dc8c359",
+           "creator": "czhual",
+           "created_at": "2021-03-03T18:58:10Z",
+           "updated_at": "2021-03-03T18:58:10Z"
+        }
+
+    """
+    url = urljoin(url, f"{URL_PATH_PREFIX}/datasets/{dataset_name}/drafts/{draft_number}")
+    return open_api_do("GET", access_key, url).json()  # type: ignore[no-any-return]
+
+
+def update_draft(
+    url: str,
+    access_key: str,
+    dataset_name: str,
+    draft_number: int,
+    *,
+    state: Optional[str] = None,
+    title: Optional[str] = None,
+    description: Optional[str] = None,
+) -> None:
+    """Execute the OpenAPI `PATCH /v1/datasets/{dataset_name}/drafts/{draft_number}`.
+
+    Arguments:
+        url: The URL of the graviti website.
+        access_key: User's access key.
+        dataset_name: Name of the dataset, unique for a user.
+        draft_number: The updated draft number.
+        state: The updated draft status which could be "CLOSED" or None.
+            Where None means no change in status.
+        title: The draft title.
+        description: The draft description.
+
+    Examples:
+        Update the title or description of the draft:
+
+        >>> update_draft(
+        ...     "https://graviti.com/",
+        ...     "ACCESSKEY-********",
+        ...     "MNIST",
+        ...     2,
+        ...     title="draft-3"
+        ... )
+
+        Close the draft:
+
+        >>> update_draft(
+        ...     "https://graviti.com/",
+        ...     "ACCESSKEY-********",
+        ...     "MNIST",
+        ...     2,
+        ...     status="CLOSED"
+        ... )
+
+    """
+    url = urljoin(url, f"{URL_PATH_PREFIX}/datasets/{dataset_name}/drafts/{draft_number}")
+    patch_data: Dict[str, Any] = {"draftNumber": draft_number}
+
+    if state:
+        patch_data["status"] = state
+    if title is not None:
+        patch_data["title"] = title
+    if description is not None:
+        patch_data["description"] = description
+
+    open_api_do("PATCH", access_key, url, json=patch_data)
+
+
+def delete_draft(
+    url: str,
+    access_key: str,
+    dataset_name: str,
+    draft_number: int,
+) -> None:
+    """Execute the OpenAPI `DELETE /v1/datasets/{dataset_name}/drafts/{draft_number}`.
```

Comment:
Deleting drafts should not be supported. There's a mistake in the documentation.